experimentalSetups/head_face-v3/camera/dragonfly2_config_right.ini

```diff
@@ -1,4 +1,6 @@
 device dragonfly2
+capabilities COLOR
+capabilities RAW

```

Comment:
Probably a find/replace error, `capabilities RAW` has to be removed here and for the other `dragonfly2`